internal/pkg/describe/backend_service.go

```diff
@@ -20,25 +21,29 @@ type BackendServiceDescriber struct {
 	service         *config.Service
 	enableResources bool
 
-	store                storeSvc
+	store                deployStoreSvc
 	svcDescriber         svcDescriber
 	initServiceDescriber func(string) error
 }
 
 // NewBackendServiceDescriber instantiates a backend service describer.
 func NewBackendServiceDescriber(app, svc string) (*BackendServiceDescriber, error) {
```

Comment:
This is def. for a separate PR but it would be cool if the constructor just had parameters for the dependencies instead of initializing them.

We can see why it's useful to do that in l.39 where previously `deploy.NewStore()` would just re-create the same `config` client so we'd have multiple location where the client initialization could fail.